src/appmixer/hubspot/bundle.json

```
@@ -64,7 +64,7 @@
             "Added caching of Contact and Deal properties, which can be further configured in the connector settings.",
             "Improved listing output variables for `GetContact` to make it consistent with the selected properties in the input."
         ],
-        "4.2.0": [
+        "4.2.1": [
             "Added support for Notes: added components `Create Note`, `Delete Note`, `Find Notes` and `Updated Note`."
```

```suggestion
            "Added support for Notes: added components `Create Note`, `Delete Note`, `Find Notes` and `Update Note`."
```
There is no "Updated Note" component, so just to avoid confusion since you are updating it now, why not also this small detail.

---

@ZbynekPelunek fixed